modules/help_editor/templates/form_edit_help_content.tpl

```diff
@@ -35,10 +28,13 @@
             <div class="col-xs-12"></div>
         {/foreach}
         <br>
-        <input class="btn btn-sm btn-primary col-sm-offset-3" name="fire_away" value="Save" type="submit" />
+        <input type="hidden" id="section" value="{$section}">
+        <input type="hidden" id="subsection" value="{$subsection}">
+        <input type="hidden" id="helpID" value="{$helpID}">
+        <input type="hidden" id="parentID" value="{$parentID}">
+        <input class="btn btn-sm btn-primary col-sm-offset-3" id="save-help" name="fire_away" value="Save" type="submit" />
         <input type="button" name="reset" value="Reset" class="btn btn-sm btn-primary" onclick="location.href='{$baseurl}/help_editor/edit_help_content/?section={$section}&subsection={$subsection}'" />
```

Comment:
This would need to be included in the javascript to prevent from doing a HTTP request (that would change the referer to this page instead of the previous)

Reply:
i also need to change the swal confirm button text for when it returns to the Help Editor module, instead of the instrument/module the help content is for

Reply:
done